invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -28,15 +28,13 @@
               <i class="eye icon"></i> {{_ ("Preview")}}
             </div>
             <p>
-              {% if record.is_published %}
+              {% if not is_draft %}
+               You are previewing a published record.
```

Comment:
It seems like if we give a `can_view` permission link for a private record to someone, then that person will see this preview banner even though they are not "previewing", but viewing...

Honestly I think the problem lies in the overlapping semantics of can_preview / can_view. The semantics should be: 
- `can_preview` is about being able to see a draft on the landing page
- `can_view` is about being able to see a record on the landing page

To me, the simple solution would be changing the below from:

```
You are previewing changes that have not yet been published.
``` 

to 

```
You are previewing changes that may not have been published yet.
```
i.e. we are not showing a visual diff, so we have no way of telling people *what* about the draft is different from the published record. At best we can tell them that some of the draft may not have been published.

As an aside, personally I am not super convinced about the need of `can_preview` if a depositor can simply publish as private and then give `can_view` link to others.

Reply:
After discussion, new issue created <https://github.com/inveniosoftware/invenio-app-rdm/issues/912>.